books/free-programming-books-langs.md

```diff
@@ -1065,6 +1066,8 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [Writing Advanced Applications for the Java 2 Platform](http://www.pawlan.com/monica/books/AdvBk.pdf) - Calvin Austin and Monica Pawlan (PDF)
 
 
+
+
```

Comment:
```suggestion
```

_"1026:1-1066:144  warning  Incorrect number of blank lines between last section and next heading  blank-lines-1-0-2  remark-lint"_: you writed two more blank lines, please read [CONTRIBUTING.md](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md#formatting) for more information

Reply:
I have made the changes